### ISSUE TITLE
Fine-tuning with my own dataset

### ISSUE THREAD

**Issue by XYQ0121** [DepthAnything/Depth-Anything-V2#31]
Hello, I noticed that the fine-tuning of depth-anything-v1 was performed using the KITTI dataset (with sparse depth maps as ground truth). However, depth-anything-v2 was fine-tuned using Virtual KITTI 2 (with dense depth maps as ground truth). When I fine-tune depth-anything-v2 using my self-collected dataset (with sparse depth maps as ground truth), there are issues with the loss. Is there any good solution for this?
![Snipaste_2024-06-26_10-47-39](https://github.com/DepthAnything/Depth-Anything-V2/assets/110807624/e447a059-8940-475d-ac95-7f5fd5a7ebaa)

**Comment by LiheYoung** [comment]
Hi, did you correctly ignore the regions without groundtruth values during loss computation?

**Comment by XYQ0121** [comment]
> Hi, did you correctly ignore the regions without groundtruth values during loss computation?

It seems that the issue with the loss function occurred because I did not set the learning rate correctly during training. However, even when I correctly set the training parameters and used the same data to fine-tune both v1 and v2, v2 performed worse than v1. Could this be because I trained using sparse depth maps?
![Snipaste_2024-07-03_10-00-30](https://github.com/DepthAnything/Depth-Anything-V2/assets/110807624/be5bff5a-97ae-4c09-9287-a0308a2f9550)
![Snipaste_2024-07-03_10-01-50](https://github.com/DepthAnything/Depth-Anything-V2/assets/110807624/3da5f14c-972d-48e6-8deb-75c262d77c4b)
![Snipaste_2024-07-03_10-01-50](https://github.com/DepthAnything/Depth-Anything-V2/assets/110807624/23b27c78-60dc-46dc-9314-52971ae68a11)

**Comment by XiaoLuoLYG** [comment]
@XYQ0121 Hi, I met the same Nan loss problem, how did you solve this. I set LR as default 0.000005

**Comment by Edric-star** [comment]
@XYQ0121 Hi there, would you mind sharing me with some of your details in fine-tuning the metric depth of depth V2? Like the amount ofyour datasets and how sparse your datasets are(did you simply set the valid_mask as gt > 0 ?).
My problems here are that my models are really hard to get a loss like 0.0***, usually 0.2-0.4, and the eval parameters are not really good as d1 - d3 from 0.6-0.8
